src/Nuget/Sarif.Multitool.nuspec

```diff
@@ -26,15 +27,14 @@
           />
     <file src="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net461\**"
           target="tools\net461"
-          exclude="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net461\Sarif*"
+          exclude="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net461\Sarif*;bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net461\WorkItems*;"
```

Comment:
>bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net461\Sarif*;bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net461\WorkItems*; [](start = 19, length = 150)

this exclusion prevents the unsigned binary from getting copied as well as the signed binary. since workitems.dll is signed, the lack of an exclusion resulted in it being duplicated in the package.